src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -1,12 +1,13 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-import { WorkAllocationTaskService } from 'src/work-allocation/services/work-allocation-task.service';
 
-import { TaskService, TaskSort } from '../../enums';
+import { TaskService, TaskSort, TaskActionIds } from '../../enums';
```

Comment:
I'd guess this will give a lint warning as they're not alphabetical. :)

Reply:
Ah yep. Done.

Reply:
Good spot!